modes/mpdel/evil-collection-mpdel.el

```diff
@@ -0,0 +1,94 @@
+;;; evil-collection-mpdel.el --- Evil bindings for mpdel -*- lexical-binding: t -*-
+
+;; Copyright (C) 2021 Ruoyu Feng
+
+;; Author: Ruoyu Feng <mail@vonfry.name>
+;; Maintainer: James Nguyen <james@jojojames.com>
+;; Pierre Neidhardt <mail@ambrevar.xyz>
+;; URL: https://github.com/emacs-evil/evil-collection
+;; Version: 0.0.4
+;; Package-Requires: ((emacs "25.1"))
+;; Keywords: evil, tools, mpd
+
+;; This file is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published
+;; by the Free Software Foundation; either version 3, or (at your
+;; option) any later version.
+;;
+;; This file is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+;;
+;; For a full copy of the GNU General Public License
+;; see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+;; Evil bindings for mpdel.
+
+;;; Code:
+
+(require 'mpdel nil t)
+(require 'evil-collection)
+
+(defvar mpdel-core-map)
+
+(defconst evil-collection-mpdel-maps '(mpdel-browser-mode mpdel-playlist-mode
```

Comment:
These should be mode maps.

Reply:
Sorry for using wrong name. These modes are passed to `evil-set-initial-state`. I will rename it to `evil-collection-mpdel-modes`

Reply:
You'll want another const holding all the relevant mode maps.

Reply:
The `mpdel-core-map` is the original one used by mpdel. It doesn't contain `mode` in its name.